### PR TITLE
feat: stage vagrant vmware page

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksvagrant-vmware-page-hashicorp.vercel.app/vagrant/downloads/vmware) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1202888288953409) 🎟️
- [Figma](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=13122%3A171549) 🎨 

## 🗒️ What

Adds the vagrant vmware install page. 

## 🛠️ How

Adjusted the current install page functions to account for the various edge cases. 

## 📸 Design Screenshots
Screen Shot 2022-09-12 at 11 38 58 AM

## 🧪 Testing

- [Visit this page ](https://dev-portal-git-ksvagrant-vmware-page-hashicorp.vercel.app/vagrant/downloads/vmware)
- Ensure the download links work as expected
- Also visit the[ install page](https://dev-portal-git-ksvagrant-vmware-page-hashicorp.vercel.app/vagrant/downloads)
- Check that the vmware merchandising slot is rendering as expected.

## 💭 Anything else?

~~I'm not sure what we want this to say? Should we remove it or just leave it to the content folks. Looks like it's auto-generated.~~ I removed this.
Screen Shot 2022-09-12 at 11 37 31 AM

### CODE REVIEW

File: src/content/vagrant/install-vmware-landing.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+	"featuredTutorialsSlugs": [
+		"vagrant/getting-started-index",
+		"vagrant/getting-started-boxes",
+		"vagrant/getting-started-networking"
+	],
+	"packageManagerOverrides": [],
```
Comment (kendallstrautman) [inline src/content/vagrant/install-vmware-landing.json:7]: Should I adjust this so the package manager codeblock doesn't render? Is it possible to do that via the content?

File: src/views/product-downloads-view/components/page-header/index.tsx [hunk 2]
```diff
@@ -9,20 +9,24 @@ import s from './page-header.module.css'
...
 	const pageTitle = isEnterpriseMode
 		? `${currentProduct.name} Enterprise Installation`
-		: `Install ${currentProduct.name}`
+		: `Install ${productName || currentProduct.name}`
+
 	const pageSubtitle = isEnterpriseMode
 		? ''
 		: getPageSubtitle({
-				productName: currentProduct.name,
+				productName: productName || currentProduct.name,
```
Comment (kendallstrautman) [inline src/views/product-downloads-view/components/page-header/index.tsx:29]: Needed a way to override the product name for this view.

File: src/pages/vagrant/downloads/index.tsx [hunk 3]
```diff
@@ -1,9 +1,30 @@
+import GetStartedCard from 'components/get-started-card'
 import vagrantData from 'data/vagrant.json'
 import { ProductData } from 'types/products'
 import ProductDownloadsView from 'views/product-downloads-view'
 import { generateGetStaticProps } from 'views/product-downloads-view/server'
 
+function VagrantDownloadsMerchandisingSlot() {
+	return (
+		<GetStartedCard
```
Comment (kendallstrautman) [inline src/pages/vagrant/downloads/index.tsx:9]: I'm going to rename this component to just 'Callout' in a follow up PR, as it's used on the vagrant landing page multiple times, not just in the getting started context.

File: src/views/product-downloads-view/index.tsx [hunk 4]
```diff
@@ -114,7 +115,10 @@ const ProductDownloadsViewContent = ({
 					<meta name="robots" key="robots" content="noindex, nofollow" />
 				</HashiHead>
 			) : null}
-			<PageHeader isEnterpriseMode={isEnterpriseMode} />
+			<PageHeader
+				isEnterpriseMode={isEnterpriseMode}
+				productName={installName}
+			/>
```
Comment (BRKalow) [inline src/views/product-downloads-view/index.tsx:121]: thought: we could simplify `PageHeader` and avoid the fallbacks if we pass in `currentProduct.name` here:
```suggestion
			<PageHeader
				isEnterpriseMode={isEnterpriseMode}
				productName={installName || currentProduct.name}
			/>
```

File: src/content/vagrant/install-vmware-landing.json (hunk 1 above)
Comment (BRKalow) [reply src/content/vagrant/install-vmware-landing.json:7]: Looks like we can use `doesNotHavePackageManagers`, per https://github.com/hashicorp/dev-portal/tree/main/src/content#5-product-install-landing-pages

Comment (kendallstrautman) [reply src/content/vagrant/install-vmware-landing.json:7]: ohhh 💯

**Review by BRKalow — APPROVED** [review]

**Review by zchsh — APPROVED** [review]
✅ 🙌 Ran through the testing steps, this looks great to me!

**Pull request merged**